Source/Client/Persistent/CaravanSplitting_Proxy.cs

```diff
@@ -0,0 +1,148 @@
+            GUI.BeginGroup(inRect);
+            Rect rect2 = inRect.AtZero();
+            DoBottomButtons(rect2);
+            Rect inRect2 = rect2;
+            inRect2.yMax -= 59f;
+            bool flag = false;
+            switch (tab)
+            {
+                case Tab.Pawns:
+                    pawnsTransfer.OnGUI(inRect2, out flag);
+                    break;
+                case Tab.Items:
+                    itemsTransfer.OnGUI(inRect2, out flag);
+                    break;
+            }
+            if (flag)
+            {
+                CountToTransferChanged();
+            }
+            GUI.EndGroup();
+        }
+
+        /// <summary>
+        /// Replaces Dialog_SplitCaravan.DoBottomButtons. 
+        /// This is a copy of the original but with the handlers for the buttons pulled out into seperate handlers.
+        /// </summary>
+        /// <param name="rect"></param>
+        public new void DoBottomButtons(Rect rect)
+        {
+            float num = rect.width / 2f;
+            Vector2 bottomButtonSize = BottomButtonSize;
+            float x = num - bottomButtonSize.x / 2f;
+            float y = rect.height - 55f;
+            Vector2 bottomButtonSize2 = BottomButtonSize;
+            float x2 = bottomButtonSize2.x;
+            Vector2 bottomButtonSize3 = BottomButtonSize;
+            Rect rect2 = new Rect(x, y, x2, bottomButtonSize3.y);
+            if (Widgets.ButtonText(rect2, "AcceptButton".Translate(), true, false, true))
+            {
+                AcceptButtonClicked();
+            }
+            float num2 = rect2.x - 10f;
+            Vector2 bottomButtonSize4 = BottomButtonSize;
+            float x3 = num2 - bottomButtonSize4.x;
+            float y2 = rect2.y;
+            Vector2 bottomButtonSize5 = BottomButtonSize;
+            float x4 = bottomButtonSize5.x;
+            Vector2 bottomButtonSize6 = BottomButtonSize;
+            Rect rect3 = new Rect(x3, y2, x4, bottomButtonSize6.y);
+            if (Widgets.ButtonText(rect3, "ResetButton".Translate(), true, false, true))
+            {
+                ResetButtonClicked();
+            }
+            float x5 = rect2.xMax + 10f;
+            float y3 = rect2.y;
+            Vector2 bottomButtonSize7 = BottomButtonSize;
+            float x6 = bottomButtonSize7.x;
+            Vector2 bottomButtonSize8 = BottomButtonSize;
+            Rect rect4 = new Rect(x5, y3, x6, bottomButtonSize8.y);
+            if (Widgets.ButtonText(rect4, "CancelButton".Translate(), true, false, true))
+            {
+                CancelButtonClicked(); 
+            }
+        }
+
+        private void AcceptButtonClicked()
+        {
+            CaravanSplittingSession.AcceptSplitSession();
+        }
+
+        private void CancelButtonClicked()
+        {
+            CaravanSplittingSession.CancelSplittingSession();
+        }
+
+        private void ResetButtonClicked()
+        {
+            SoundDefOf.Tick_Low.PlayOneShotOnCamera(null);
```

Comment:
The parameter `onlyThisMap` has the same default value.

Reply:
Sure I'll change this.